packages/order-utils/src/types.ts

```diff
@@ -37,6 +37,18 @@ export interface FindOrdersThatCoverMakerAssetFillAmountOpts {
     slippageBufferAmount?: BigNumber;
 }
 
+/**
+ * remainingFillableMakerAssetAmount: An array of BigNumbers corresponding to the `orders` parameter.
+ * You can use `OrderStateUtils` `@0x/order-utils` to perform blockchain lookups for these values.
+ * Defaults to `makerAssetAmount` values from the orders param.
+ * slippageBufferAmount: An additional amount of makerAsset to be covered by the result in case of trade collisions or partial fills.
+ * Defaults to 0
+ */
+export interface FindOrdersThatCoverTakerAssetFillAmountOpts {
```

Comment:
nit: a bit odd that this is named like a function with `Find`

Reply:
Yeah, not a fan of it but the convention for the opts for the other functions fall under this pattern, open to changing this and the convention.